inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/source/sortsdk/SortSdkSource.java

```diff
@@ -249,14 +257,18 @@ private Map<String, String> getSortClientConfigParameters() {
      */
     @Override
     public void stopConsumer() {
-        sortClient.getConfig().setStopConsume(true);
+        for (SortClient sortClient : sortClients) {
+            sortClient.getConfig().setStopConsume(true);
+        }
     }
 
     /**
      * recoverConsumer
      */
     @Override
     public void recoverConsumer() {
-        sortClient.getConfig().setStopConsume(false);
+        for (SortClient sortClient : sortClients) {
+            sortClient.getConfig().setStopConsume(false);
+        }
     }
-}
+}
```

Comment:
No need to delete the blank line at the end of the file, otherwise there is a symble in GitHub.
![image](https://user-images.githubusercontent.com/31994335/188426080-75c6d823-e59b-473c-a292-e0b2877e94bc.png)

Reply:
fixed, thx